src/frontend/next/src/components/header/Header.js

```diff
@@ -0,0 +1,94 @@
+import styles from '../../styles/Header.module.css';
+import Link from 'next/link';
+import React, { useState } from 'react';
+
+export default function Header() {
+  const [open, setOpen] = useState(false);
```

Comment:
NOTE: this will work due to inferred typing on `false` being `boolean`, but you can also be more explicit by doing:

```ts
const [open, setOpen] = useState<boolean>(false);
```

Reply:
@humphd <boolean> requires the Header.js to be a TypeScript file.

Reply:
Yes, we are porting our frontend to NEXT.JS and TS. 
So what you need to do is create a TS version of the`Header`.

Reply:
@PedroFonsecaDEV oh, okay. Will do. Is there anything else I need to know about the process of changing js file to TS?

Reply:
Try to follow the same pattern as the other TS components. 
We are using these docs as a reference: https://github.com/Seneca-CDOT/telescope/issues/1557

Again, if you need help, let me know. We can schedule a meet on Teams.